packages/lesswrong/lib/sql/Type.ts

```diff
@@ -0,0 +1,255 @@
+import { getCollection } from "../vulcan-lib/getCollection";
+import GraphQLJSON from 'graphql-type-json';
+import SimpleSchema from "simpl-schema";
+import { ID_LENGTH } from "../random";
+
+const forceNonResolverFields = ["contents", "moderationGuidelines", "customHighlight", "originalContents"];
```

Comment:
Why do these fields need to be handled separately, and couldn't we add `resolveAs` to them in some way? I think I could easily call a field `originalContents` in future and then spend forever trying to track down the source of the bug